src/js/xr/src/use-interactions-manager.js

```diff
@@ -855,7 +863,6 @@ const useInteractionsManager = ({
             object.userData.staticRotation = null
           }
 
-          controller.userData.selectOffset = null
```

Comment:
`selectOffset` is no longer cleared when we're done?

Reply:
We can keep but the thing is that objects don't use it anymore cause objects added to controller when they are selected. So there is no need for offset and if we apply it to object through our formula with the cursor we will get doubled rotation cause now object follows the rotation of the controller and cursor follows the rotation of the controller.

Reply:
Oh ok. So we could remove all references to selectOffset now?

Reply:
Well, character still use it so I don't think we can completely remove it yet